feature-libs/checkout/base/core/utils/utils.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { Address } from '@spartacus/core';
+import { getAddressNumbers } from './utils';
+
+describe('Checkout utils', () => {
+  describe('getAddressNumbers', () => {
+    const mockAddress: Address = {
+      firstName: 'Mock',
+      lastName: 'Address',
+      phone:'12345',
+      cellphone:'67890'
+    };
+
+    it('should return phone number and mobile number', () => {
+      expect(getAddressNumbers(mockAddress, 'P', 'M')).toEqual(`P: ${mockAddress.phone}
+      M: ${mockAddress.cellphone}`);
```

Comment:
```suggestion
      expect(getAddressNumbers(mockAddress, 'P', 'M')).toEqual(`P: ${mockAddress.phone}\nM: ${mockAddress.cellphone}`);
```
or
```suggestion
      expect(getAddressNumbers(mockAddress, 'P', 'M')).toEqual(`P: ${mockAddress.phone}\n\M: ${mockAddress.cellphone}`);
```
if prettier is being uppity.